VanDerPol/FMI1ME.xml

```diff
@@ -15,19 +15,19 @@
     <ScalarVariable name="time" valueReference="0" causality="internal" variability="continuous" description="Simulation time">
       <Real/>
     </ScalarVariable>
-    <ScalarVariable name="x0" valueReference="1" causality="output" description="the first state">
+    <ScalarVariable name="x0" valueReference="1" causality="output" description="Position">
```

Comment:
x0 is not a position

Reply:
We changed it in #259 accordingly. Als https://en.wikipedia.org/wiki/Van_der_Pol_oscillator mentions that x0 is the position corórdinate.